js/livecoin.js

```diff
@@ -231,7 +231,7 @@ module.exports = class livecoin extends Exchange {
         let currencies = [
             { 'id': 'USD', 'code': 'USD', 'name': 'US Dollar' },
             { 'id': 'EUR', 'code': 'EUR', 'name': 'Euro' },
-            { 'id': 'RUB', 'code': 'RUB', 'name': 'Russian ruble' },
+            { 'id': 'RUB', 'code': 'RUR', 'name': 'Russian ruble' },
```

Comment:
We're using RUB for Russian Rubles, because of this: https://github.com/ccxt/ccxt/issues/2481

Reply:
Sorry - got id and code mixed up. So code is the one common to all exchanges and id is specific to each exchange right?

Reply:
Yep, exactly ) 
- id = exchange-specific
- `code` = unified
- `symbol` = unified
- `base` = unified
- `quote` = unified
- `baseId` = exchange-specific
- `quoteId` = exchange-specific